source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/ContainerRenderer.java

```diff
@@ -136,6 +137,7 @@ public static void ApplyBleed(CollectionTypeElement collectionElement, ViewGroup
         {
             int padding = Util.dpToPixels(context, hostConfig.GetSpacing().getPaddingSpacing());
             ViewGroup.MarginLayoutParams layoutParams = (ViewGroup.MarginLayoutParams) collectionElementView.getLayoutParams();
+            // TODO: Check RTL support
```

Comment:
layoutParam doesn't have the direction neutral params for setting direction like view_start?

Reply:
Checked, works